UnitsNet.Tests/UnitParserTests.cs

```diff
@@ -137,5 +137,20 @@ public void Parse_MappedCustomUnit()
 
             Assert.Equal(HowMuchUnit.Some, parsedUnit);
         }
+        
+        [Fact]
+        public void TryParse_WithNullAbbreviation_ReturnsFalse()
+        {
+            UnitParser unitParser = UnitsNetSetup.Default.UnitParser;
+            Assert.Multiple(() =>
+            {
+                var success = unitParser.TryParse(null, out LengthUnit unit);
+                Assert.False(success);
+            }, () =>
+            {
+                var success = unitParser.TryParse(null, typeof(LengthUnit), out Enum? _);
+                Assert.False(success);
+            });
+        }
```

Comment:
Are you Ok with this type of testing? A lot of my v6 tests use the same convention (_merging only when **the method name is the same**_).

Reply:
Absolutely, I have a very low bar for testing schemes 😄 as long as they do the job and the name of the test makes sense and kinda follows the other tests in the same area.

Reply:
Just to mention, we're using [FluentAssertions](https://github.com/fluentassertions/fluentassertions) at work and I do love that. 

The syntax is a breeze, assertion messages way more readable and in this case you would do `using var _ = new AssertionScope();` before all the assertions.

I wouldn't mind refactoring all tests over to it if I found surplus energy and time. _wink wink, nudge nudge_ 😆

Reply:
Don't know if you've got an AI agent- but given a little bit of preparing the right prompt- it should be able to reformat a whole class without much trouble...
If you want to start us up with an example class, I might try it out (_one day_).